### PR TITLE
Sendgrid Cloud Extract documentation

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Added Sendgrid documentation

## Type of documentation

- [x] New documentation
- [ ] Documentation update
- [ ] Hotfix

## Notion ticket link

> [Sendgrid documentation](https://www.notion.so/rudderstacks/Sendgrid-Cloud-Extract-Source-dc02f90fe682445ab5c81c2d4bfb750f)

### CODE REVIEW

File: cloud-extract-sources/destination-settings.mdx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+---
+
+# Destination Settings
+
+This guide details the common destination settings to be configured while setting up your Cloud Extract sources.
+
+These settings are:
+
+- Table prefix
+- Sync schedule settings
```
Comment (ishachopra0401) [inline cloud-extract-sources/destination-settings.mdx:14]:
```suggestion
- Schedule settings
```
As displayed in the screenshot

File: cloud-extract-sources/destination-settings.mdx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+<img src="../assets/cloud-extract-sources/sendgrid-connection-settings-2.png" alt="Sendgrid destination settings" />
+
+## Table prefix
+
+RudderStack uses your specified table prefix to create a table in your data warehouse and load all your Sendgrid data into it.
```
Comment (ishachopra0401) [inline cloud-extract-sources/destination-settings.mdx:20]:
```suggestion
RudderStack uses your specified table prefix to create a table in your data warehouse and load all the destination data into it.
```

File: cloud-extract-sources/destination-settings.mdx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+<div class="infoBlock">
+
+If you do not specify this field, RudderStack sets the source name as the table name, by default.
+</div>
+
+## Sync schedule
```
Comment (ishachopra0401) [inline cloud-extract-sources/destination-settings.mdx:27]:
```suggestion
## Schedule settings
```

File: cloud-extract-sources/destination-settings.mdx (hunk 2 above)
Comment (ameypv-rudder) [reply cloud-extract-sources/destination-settings.mdx:20]: actually it should be the source data, i.e. data from the cloud extract source

Comment (ameypv-rudder) [reply cloud-extract-sources/destination-settings.mdx:20]: tweaked your suggestion accordingly.

File: cloud-extract-sources/sendgrid.mdx [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+For more information on the required permissions for the API key, refer to the <a href="#faq">FAQ</a> section below.
+</div>
+
+### Destination settings
+
+The following settings specify how RudderStack sends the data ingested from Sendgrid to the connected warehouse destination:
+
+- **Table prefix**: RudderStack uses this prefix to create a table in your data warehouse and load all your Sendgrid data into it.
+- **Schedule Settings**: Select your schedule type. RudderStack gives you three options to ingest the data from Sendgrid:
```
[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply cloud-extract-sources/sendgrid.mdx:46]: this is fine, we are elaborating in the Destination Settings doc anyway.

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**